app/build.gradle.kts

```
@@ -68,7 +68,7 @@ android {
         buildConfigField(
             type = "String",
             name = "CI_INFO",
-            value = "\"${ciProperties.getOrDefault("ci.info", "local")}\""
+            value = "${ciProperties.getOrDefault("ci.info", "local")}"
```

The property value is being written with surrounding double quotes but we still need to make sure this default value has them.

```suggestion
            value = "${ciProperties.getOrDefault("ci.info", \""local\"")}"
```